espresso/docker/op-stack/Dockerfile

```diff
@@ -1,12 +1,12 @@
 # OP Stack Dockerfile, simplified from ops/docker/op-stack-go/Dockerfile
 
 # Build arguments
-ARG TARGET_BASE_IMAGE=ubuntu:22.04
+ARG TARGET_BASE_IMAGE=alpine:3.22
```

Comment:
Switched target to alpine so that we don't have a zoo of base images anymore (we've been using alpine for golang and rust builders). Chose alpine because it's quite a bit smaller.